components/brave_vpn/brave_vpn_utils.cc

```diff
@@ -5,13 +5,34 @@
 
 #include "brave/components/brave_vpn/brave_vpn_utils.h"
 
+#include "base/command_line.h"
 #include "base/feature_list.h"
 #include "brave/components/brave_vpn/features.h"
+#include "brave/components/brave_vpn/switches.h"
+#include "brave/components/brave_vpn/url_constants.h"
 
 namespace brave_vpn {
 
 bool IsBraveVPNEnabled() {
   return base::FeatureList::IsEnabled(brave_vpn::features::kBraveVPN);
 }
 
+std::string GetManageUrl() {
+  auto* cmd = base::CommandLine::ForCurrentProcess();
+  if (!cmd->HasSwitch(brave_vpn::switches::kBraveVPNAccountHost))
```

Comment:
Should this maybe be checking official build? I think most cases we'd want to use dev basically

Reply:
Yeah, using official build flag here seems good.

Reply:
Updated.